exir/passes/external_constants_pass.py

```diff
@@ -74,3 +86,34 @@ def external_mutable_weights_pass(
                     node.meta["constant_tag"] = "_default_external_constant"
                     mutated = True
     return PassResult(gm, mutated)
+
+
+def delegate_external_constants_pass(
+    gm: GraphModule,
+    ep: ExportedProgram,
+    filter_fn: Optional[Callable[[torch.fx.Node], str]] = None,
+) -> PassResult:
+    """
+    Tag external constants before to_backend. Tagged constants will be saved
+    to an external file.
```

Comment:
This function doesn't know how the tags will be used.

```suggestion
    Tag external constants before to_backend.
```

Reply:
thanks!